db/migrations/v2.3.0.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE user ADD alert BOOLEAN NOT NULL AFTER password;
```

Comment:
not sure if `BOOLEAN` type is new, check compatibility with versions of mysql/mariadb listed in wiki